utils/frame/try-runtime/cli/tests/follow_chain.rs

```diff
@@ -0,0 +1,65 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+#![cfg(unix)]
+
+#[cfg(feature = "try-runtime")]
+mod tests {
+	use assert_cmd::cargo::cargo_bin;
+	use regex::Regex;
+	use std::{
+		process::{self},
+		time::Duration,
+	};
+	use substrate_cli_test_utils as common;
+	use tokio::process::{Child, Command};
+
+	#[tokio::test]
+	async fn follow_chain_works() {
```

Comment:
now that we have the infra, should we make tests for all commands? can be an issue that you either do yourself, or mentor.

Reply:
Yes, I think we should have tests for subcommands too; otherwise they'll just accidentally break sooner or later.

Reply:
Absolutely, it's tracked in this issue: https://github.com/paritytech/substrate/issues/13796. I wanted to test just one command and get the approach reviewed before adding tests for the rest of them. 

There're issues reported with some of the other `try-runtime` commands that I'm close to getting started on, I'll utilise this test infra while debugging those and leave the tests in the fix PR.